Mage/CommandCenter/TransportClient/CommandHTTPClient.cs

```diff
@@ -87,12 +87,12 @@ public class CommandHTTPClient : CommandTransportClient {
 			// Check if there was a transport error
 			if (requestError != null) {
 				string error = "network";
-				if (requestError is WebException) {
-					WebException webException = requestError as WebException;
-					HttpWebResponse webResponse = webException.Response as HttpWebResponse;
-					if (webResponse != null && webResponse.StatusCode == HttpStatusCode.ServiceUnavailable) {
-						error = "maintenance";
-					}
+
+				// On error
+				var wwwError = requestError as HTTPRequestException;
```

Comment:
technically this is not a wwwError, its a httprequestException
also what happens if that cast fails?
i think this is why its wrapped in a condition
also please copy the style from here
https://github.com/AlmirKadric/mage-sdk-unity/blob/feature/HTTPRequestWWWW/Mage/MessageStream/TransportClient/LongPolling.cs#L105